### PR TITLE
finished basics of rating breakdown

### PR DESCRIPTION
**Pull request by gusfel**

### CODE REVIEW

**Review by jincheng411 — APPROVED** [review]

**Review by laenalea — APPROVED** [review]
Great progress, really impressed by the clever rendering of data reliant on async functions.

File: Reviews/src/Components/RatingBreakdown/AvgRating.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const AvgRating = ({ ratings }) => {
+  let reviewCount = 0;
+  let reviewTotal = 0;
+
+  Object.keys(ratings).forEach(key => {
+    reviewCount += Number(ratings[key]);
+    reviewTotal += Number(key) * Number(ratings[key]);
+  });
+
```
Comment (momo1113) [inline Reviews/src/Components/RatingBreakdown/AvgRating.jsx:12]: Nice function to use 👍

**Review by momo1113 — APPROVED** [review]

File: Reviews/src/Components/RatingBreakdown/AvgRating.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AvgRating = ({ ratings }) => {
+  let reviewCount = 0;
+  let reviewTotal = 0;
+
+  Object.keys(ratings).forEach(key => {
```
Comment (momo1113) [inline Reviews/src/Components/RatingBreakdown/AvgRating.jsx:8]: Nice function to use 👍

**Review by momo1113 — APPROVED** [review]

**Comment by momo1113** [comment]
Lots of progress, nice to put all the sub-components into its owe module.

**Pull request merged**